form/form.go

```diff
@@ -59,157 +117,95 @@ type PromptingFiller struct {
 // validated against the field's type. If the returned value does not
 // validate correctly it will be prompted again up to MaxTries before
 // giving up.
-func (f *PromptingFiller) Fill(form Form) (map[string]interface{}, error) {
+func (f IOFiller) Fill(form Form) (map[string]interface{}, error) {
+	if f.MaxTries == 0 {
+		f.MaxTries = 3
+	}
 	if form.Title != "" {
```

Comment:
Perhaps you should wait to print the title until after you've verified the checkers. Also add a test where you have a form title and invalid fields?